### ISSUE TITLE
Codes and weights for other tasks (detection, segmentation, etc...)

### ISSUE THREAD

**Issue by ieellee** [LeapLabTHU/ViTTT#4]
@tian-qing001 Hi, Dongchen. I would like to inquire if there are any plans to release the code and weights for different loss function calculations and tasks for ViTTT. Your work is very solid! I want to study more about ViTTT.

**Comment by tian-qing001** [comment]
Hi @ieellee, thanks for your recognition. We will release the code and pretrained weights for other tasks.

**Comment by tian-qing001** [comment]
Hi @ieellee, the full code has been released.
